language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/util/SnippetGenerator.java

```diff
@@ -1662,6 +1662,19 @@ public static SnippetBlock getServiceKeywordSnippet() {
                 "service", ItemResolverConstants.KEYWORD_TYPE, Kind.KEYWORD);
     }
 
+    /**
+     * Get Service Initializer Snippet Block.
+     *
+     * @return {@link SnippetBlock}     Generated Snippet Block
+     */
+    public static SnippetBlock getServiceInitFunctionSnippet() {
+        String snippet = "function init() {" + CommonUtil.LINE_SEPARATOR + "\t${2}" +
+                CommonUtil.LINE_SEPARATOR + "}";
+        return new SnippetBlock(ItemResolverConstants.SERVICE_INITIALIZER_TYPE,
+                generateFilterText(Arrays.asList("init", ItemResolverConstants.FUNCTION)), snippet,
+                ItemResolverConstants.SNIPPET_TYPE, Kind.SNIPPET);
```

Comment:
We have an already defined init function.  Shall we use that? In the already defined one, we have function arguments added. Let's remove that and modify to reuse here as well.